resource-sharing-partners/src/test/java/no/sikt/rsp/ResourceSharingPartnerTest.java

```diff
@@ -69,25 +76,32 @@ public class ResourceSharingPartnerTest {
     public static final String TIDEMANN = "Tidemann";
     public static final String OTHER = "other";
 
+    private static final String ILL_SERVER_ENVIRONMENT_NAME = "ILL_SERVER";
+    private static final String ILL_SERVER_ENVIRONMENT_VALUE = "eu01.alma.exlibrisgroup.com";
+    public static final int ILL_SERVER_PORT = 9001;
+
+    private static final String NNCIP_SERVER = "http://nncipuri.org";
     private ResourceSharingPartnerHandler resourceSharingPartnerHandler;
     public static final Context CONTEXT = mock(Context.class);
 
     private FakeS3Client s3Client;
     private S3Driver s3Driver;
 
+    private final Environment mockedEnvironment = mock(Environment.class);
```

Comment:
Environment kan settes i build.gradle for test, da slipper vi bruke mockedEnvironment. (Forutsatt at vi ikke skal teste med forskjellige verdier da :angel: )

Reply:
Vi var nødt til å ta ibruk mockEnvironment pga. host post måtte sendes inn programmatisk.